keras_eval/utils.py

```diff
@@ -543,3 +543,20 @@ def results_differential(dataframes, mode='average', round_decimals=4, save_csv_
         differential_dataframe.to_csv(save_csv_path, float_format='%.' + str(round_decimals) + 'f', index=False)
 
     return differential_dataframe
+
+
+def check_result_type(result_csv, individual):
+    '''
+    Checks if the file type is of the required format i.e. individual or average metrics
```

Comment:
Can you elaborate more this function description?  What file (I assume `results_csv`)?